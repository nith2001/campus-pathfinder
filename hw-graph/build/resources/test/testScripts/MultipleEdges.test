# Create a graph
CreateGraph graph1

# Add 2 nodes
AddNode graph1 n1
AddNode graph1 n2

# Print the nodes in the graph
ListNodes graph1

# Add 4 edges pointing from one node to the other
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2
AddEdge graph1 n1 n2 e3
AddEdge graph1 n1 n2 e4

# Print the children of the node in the graph
ListChildren graph1 n1
