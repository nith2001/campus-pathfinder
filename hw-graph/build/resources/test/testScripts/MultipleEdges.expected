# Create a graph
created graph graph1

# Add 2 nodes
added node n1 to graph1
added node n2 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2

# Add 4 edges pointing from one node to the other
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1
added edge e3 from n1 to n2 in graph1
added edge e4 from n1 to n2 in graph1

# Print the children of the node in the graph
the children of n1 in graph1 are: n2(e1) n2(e2) n2(e3) n2(e4)
